AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java

```diff
@@ -0,0 +1,74 @@
+package Ex2Upgrade;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.support.How;
+
+public class Ex2UpgradeMain {
+    public static WebDriver driver;
+    public static void main (String []args) throws InterruptedException {
+
+        openBrowser("Chrome");
+        visitWebsite("https://the-internet.herokuapp.com/login");
+
+        fill(getElement(How.NAME,"username"),"tomsmith");
+        fill(getElement(How.NAME,"password"),"SuperSecretPassword!");
+
+        login(How.TAG_NAME,"button");
+        Thread.sleep(2000);
+        logout(How.CLASS_NAME,"button"); //how to add full class name "button secondary radius"
+
+        Thread.sleep(2000);
+        driver.quit();
+    }
+
+    public static WebElement getElement(How how, String locator){
+        return driver.findElement(how.buildBy(locator));
+    }
+
+    public static void login(How how, String locator){
+        driver.findElement(how.buildBy(locator)).submit();
```

Comment:
You can use getElement menthod here.
Ex. driver.getElement(how, locator).submit();